cmd/clusterctl/validation/validate_pods_test.go

```diff
@@ -24,7 +24,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func podWithStatus(podName, namespace string, podPhase corev1.PodPhase, containerReadyStatus bool) corev1.Pod {
+func podWithStatus(podName, namespace string, podPhase corev1.PodPhase, containerReadyStatus bool) corev1.Pod { //nolint:unparam
```

Comment:
what does this say?

Reply:
`podWithStatus` always takes same value, `test-namespace`